inc/admin.php

```diff
@@ -83,6 +83,11 @@ function __construct() {
 				add_filter( 'display_post_states', array( $this, 'add_panels_post_state' ), 10, 2 );
 			}
 		}
+
+		// Compatibility with AMP plugin
+		if ( function_exists( 'amp_bootstrap_plugin' ) ) {
+			require_once plugin_dir_path( __FILE__ ) . '../compat/amp.php';
```

Comment:
We should try keep all our compat includes in the same place. This is where we're adding another one - https://github.com/siteorigin/siteorigin-panels/blob/develop/siteorigin-panels.php#L197

What do you think about having a dedicated function in the `SiteOrigin_Panels` class that's run with `plugins_loaded` that does all our compat includes? Something like `init_compat`?

Reply:
Done. Do you want to move the SEO compatibility and Block Editor detect code (it's only there to load compatibility code) there also? If it's a yes for Yoast, I'll need to move `enqueue_seo_compat` to `siteorigin-panels.php`.